dockers/docker-base-buster/Dockerfile.j2

```diff
@@ -7,6 +7,9 @@ FROM multiarch/debian-debootstrap:arm64-buster
 FROM debian:buster
 {% endif %}
 
+RUN echo "deb [arch=amd64] http://packages.trafficmanager.net/debian/debian buster main contrib non-free" >> /etc/apt/sources.list && \
+    echo "deb [arch=amd64] http://packages.trafficmanager.net/debian/debian buster-updates main contrib non-free" >> /etc/apt/sources.list
+
```

Comment:
Why you need create this file?
Below there is a line
```
COPY ["sources.list", "/etc/apt/sources.list"]
```

Reply:
Fixed, it should be added in source.list.